hack/util.sh

```diff
@@ -96,7 +96,7 @@ function start_os_server {
       --loglevel=4 \
       1>&2 2>"${BASETMPDIR}/openshift.log" &
   else
-    sudo env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
+    env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
```

Comment:
`sudo` is required to be able to make services work correctly, so without privileged execution, the value of starting the server is severely limited.  I thought that @smarterclayton had some bash insanity that got the job done.